content/docs/user-guide/components/reference/gradient-modifiers/gradient-mixer.md

```diff
@@ -16,37 +16,61 @@ The **Gradient Mixer** component generates a new gradient by blending a number o
+| **Layers - Gradient** | Refer to the [Gradient properties](#gradient-properties) section that follows. | | |
 
 ### Gradient properties
 
 ![Gradient properties](/images/user-guide/components/reference/vegetation-modifiers/gradient-properties.png)
 
 | Property | Description | Values | Default |
 |-|-|-|-|
-| **Gradient Entity Id** | Sets an entity with an active **Gradient** component. | EntityId | None |
+| **Gradient Entity Id** | Sets an entity with an active **Gradient** or **Gradient Modifier** component. | EntityId | None |
 | **Opacity** | Sets the opacity of the input gradient. | Float: 0.0 - 1.0 | `1.0` |
 | **Invert Input** | Inverts the values of the input gradient. | Boolean | `Disabled` |
-| **Preview (Input)** | Displays the gradient provided by the entity set in **Gradient Entity Id**. |  |  |
+| **Preview (Input)** | Displays the gradient provided by the entity set in **Gradient Entity Id** with the following gradient properties applied. |  |  |
 | **Enable Transform** | If `Enabled`, the translation, scale, and rotation of the input gradient may be modified. | Boolean | `Disabled` |
 | **Translate** | Sets the translation of the input gradient. | Vector3: -Infinity to Infinity | X:`0.0`, Y:`0.0`, Z:`0.0` |
 | **Scale** | Sets the scale of the input gradient. | Vector3: 0.0001 to Infinity | X:`1.0`, Y:`1.0`, Z:`1.0` |
 | **Rotate** | Sets the rotation of the input gradient. | Vector3: -Infinity to Infinity | X:`0.0`, Y:`0.0`, Z:`0.0` |
-| **Enable Levels** | If `Enabled`, the input and output values of the input gradient may be modified. | Boolean | `Disabled` |
-| **Input Mid** | Sets the median value of the input gradient. | Float: 0.0 - 1.0 | `1.0` |
-| **Input Min** | Sets a minimum value for the input gradient. | Float: 0.0 - 1.0 | `0.0` |
-| **Input Max** | Sets a maximum value for the input gradient. | Float: 0.0 - 1.0 | `1.0` |
+| **Enable Levels** | If `Enabled`, the input and output values of the input gradient may be modified by the following properties. | Boolean | `Disabled` |
+| **Input Mid** | Remaps the midpoint of values greater than `0` and less than `1` in the input gradient. | Float: 0.0 - 1.0 | `1.0` |
+| **Input Min** | Remaps the input gradient so that it lies between **Input Min** and **Input Max**.  All output gradient values less than or equal to **Input Min** are set to `0`. | Float: 0.0 - 1.0 | `0.0` |
+| **Input Max** | Remaps the input gradient so that it lies between **Input Min** and **Input Max**.  All output gradient values less than or equal to **Input Max** are set to `1`. | Float: 0.0 - 1.0 | `1.0` |
 | **Output Min** | Sets a minimum value for the output gradient. | Float: 0.0 - 1.0 | `0.0` |
```

Comment:
```suggestion
| **Output Min** | Sets a minimum value for the output gradient. Remaps the input gradient so that it lies between **Output Min** and **Output Max**. | Float: 0.0 - 1.0 | `0.0` |
```